### DISCUSSION TITLE
Release 1.22.0 for M1 macs

### DISCUSSION THREAD

**Discussion by hkiel** [OpenModelica/OpenModelica#11543] [General]
I see nightly builds on https://build.openmodelica.org/omc/builds/mac/pkg/arm64/nightly/ for M1 macs.

@mahge Will there also be a build of the 1.22.0 release and possibly a short description on how to install it correctly? (needed dependencies etc.)
It would be good to have the release for macOS again, as I have several students with recent (M1) macBooks in my course.

**Comment by casella** [comment]
@mahge can you comment on that? I understood we had given up providing pre-compiled macOS releases.
